nncf/torch/graph/operator_metatypes.py

```diff
@@ -48,6 +48,7 @@ class PTOperatorMetatype(OperatorMetatype):
     """
 
     external_op_names: List[str] = []
+    num_expected_input_edges = None
```

Comment:
Why `None` and not at least 1?

Reply:
For an abstract metatype we don't have right value for the `num_expected_input_edges` (it could be dynamic, concat node, for example), so I chosen None as a default